issue_18/RESULTS.md

```diff
@@ -19,11 +19,14 @@ Only `x85_64` results are included at this time.
 | Rocky-8.4-x86_64-dvd1.iso       |   minimal              | media             | VirtualBox VM      | 0                 | 0                   | 0                    | 375             |
 | Rocky-8.4-x86_64-dvd1.iso       |   minimal              | media / efi       | VirtualBox VM      | 0                 | 0                   | 0                    | 412             |
 | | | | |
+| Rocky-8.4-aarch64-boot.iso      | \* graphical-server    | network           | Libvirt VM         | 9                 | 0                   | 0                    | 1239            |
 | Rocky-8.4-x86_64-boot.iso       | \* graphical-server    | network           | VirtualBox VM      | 0                 | 0                   | 0                    | 357             |
 | Rocky-8.4-x86_64-minimal.iso    |   graphical-server     | network           | VirtualBox VM      | 10                | 0                   | 0                    | 1366            |
```

Comment:
I counted 9 `Enabled Modules` rather than 10 for the above Rocky-8.4-x86_64-minimal.iso. I think this may be a mistake

Reply:
Verified. Error in command line to collect the results was counting the line starting with "Hint:" Table entries adjusted.